mv/plan/src/main/spark2.3/org/apache/carbondata/mv/plans/modular/ExpressionHelper.scala

```diff
@@ -46,5 +46,4 @@ object ExpressionHelper {
   def getTheLastQualifier(reference: AttributeReference): String = {
     reference.qualifier.head
   }
-
 }
```

Comment:
Please revert this class changes, not needed

Reply:
needed due to scalastyle